### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by sandabu**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 [English](./release-notes.en.md)
 
+## [Version 1.2.0](https://github.com/skyway/skyway-android-sdk/releases/tag/v1.2.0)
+
+### Added
+
+- MediaConnection, DataConnectionに接続中の統計情報を取得する`getStats`を追加しました
```
Comment (salmoncode) [inline release-notes.md:9]:
```suggestion
- `MediaConnection`, `DataConnection`に接続中の統計情報を取得する`getStats`を追加しました
```

File: release-notes.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+
+### Fixed
+
+- 同じ`MeshRoom`に対して多数のユーザが同時に`Peer.joinRoom`した際に、エラーが発生する不具合を修正しました
+
+### Modified
+
+- MeshRoom, SFURoomの`send`関数において、送信データ量の上限を20MBに変更しました
```
Comment (salmoncode) [inline release-notes.md:17]:
```suggestion
- `MeshRoom`, `SFURoom`の`send`関数において、送信データ量の上限を20MBに変更しました
```

Comment (ShinyaFUKUMOTO) [inline release-notes.md:17]:
```suggestion
- `SKWMeshRoom`, `SKWSFURoom`の`send`関数において、送信可能なデータのサイズ上限を20MBに変更しました
```

**Comment by salmoncode** [comment]
Is the change about modified for jp reflected?

**Comment by sandabu** [comment]
> Is the change about modified for jp reflected?

You mentioned this comment?
https://github.com/skyway/skyway-android-sdk/pull/31/files/faf01faaec6bd7d9cb750b036addb7c9a30f52e0#r416410577

No, I forget it. Thanks.

File: release-notes.md [hunk 3]
```diff
@@ -2,6 +2,20 @@
...
+
+- `MediaConnection`, `DataConnection`に接続中の統計情報を取得する`getStats`を追加しました
+
+### Fixed
+
+- 同じ`MeshRoom`に対して多数のユーザが同時に`Peer.joinRoom`した際に、エラーが発生する不具合を修正しました
+
+### Modified
+
+- MeshRoom, SFURoomの`send`関数において、送信可能なデータのサイズ上限を20MBに変更しました
```
Comment (salmoncode) [inline release-notes.md]: Only here is not fixed
```suggestion
- `MeshRoom`, `SFURoom`の`send`関数において、送信可能なデータのサイズ上限を20MBに変更しました
```

**Pull request merged**